.github/workflows/fuzzer-tester-prepare/action.yml

```diff
@@ -12,7 +12,7 @@ runs:
       with: { shared-key: "${{ runner.os }}-shared-fuzzer-cache" }
     - name: Install fuzzers deps
       shell: bash
-      run: sudo apt-get update && sudo apt-get install -y nasm nlohmann-json3-dev gcc-aarch64-linux-gnu g++-aarch64-linux-gnu gcc-mipsel-linux-gnu g++-mipsel-linux-gnu gcc-powerpc-linux-gnu g++-powerpc-linux-gnu libc6-dev-i386-cross libc6-dev libc6-dev-i386 lib32gcc-11-dev lib32stdc++-11-dev libgtk-3-dev pax-utils python3-msgpack python3-jinja2 
+      run: sudo apt-get update && sudo apt-get install -y nasm nlohmann-json3-dev gcc-aarch64-linux-gnu g++-aarch64-linux-gnu gcc-mipsel-linux-gnu g++-mipsel-linux-gnu gcc-powerpc-linux-gnu g++-powerpc-linux-gnu libc6-dev-i386-cross libc6-dev libc6-dev-i386 lib32gcc-11-dev lib32stdc++-11-dev libgtk-3-dev pax-utils python3-msgpack python3-jinja2 g++-mingw-w64
```

Comment:
@addisoncrump 
we need this?
but it is strange that it fixed your problem. this action file is not used by windows vm at all

Reply:
(including frida_gdi_plus ci that failed before this change of yours)

Reply:
i understand that build is also running on ubuntu but it's useless.
we removed it in #2949 

can you delete it? 

and after we merge #2949 then that should be gone. 
or you can just ignore that error we'll remove that anyway